.github/workflows/java-checkstyle.yml

```diff
@@ -5,10 +5,6 @@ on:
     branches:
       - master
     paths:
-      # We also trigger on non-java code changes since this triggers sonarcloud scanner after
```

Comment:
Unrelated to this PR? Or at least I can't see how it is related.

Reply:
huh... seems I forgot to rebase on master

Reply:
gone after a rebase